xml/planning-architecture-example_configurations.xml

```diff
@@ -68,6 +68,24 @@
     </para>
    </listitem>
   </itemizedlist>
+
+  <para>
+   &o_iron; multi-tenancy feature uses &o_netw; to manage the tenant
```

Comment:
"*The* Ironic multi-tenancy feature ..." ..? [NB: i have not checked the definition of \&o_iron;, if the *The* makes no sense, just ignore me on this one.]

Reply:
I choose to ignore you on this one. My warm feelings towards you remain unchanged, though.